syz-verifier/main.go

```diff
@@ -81,7 +81,7 @@ type poolInfo struct {
 	Reporter report.Reporter
 	//  vmRunners keeps track of what programs have been sent to each Runner.
 	//  There is one Runner executing per VM instance.
-	vmRunners map[int][]*progInfo
```

Comment:
This commit has nothing to do with the crashes commit, right? At least the connection isn't obvious (nor does commit message say).

Reply:
Not strictly connected, it makes the architecture simpler. Until now we would only remove elements from `vmRunners` (which keeps track of what programs have been sent to what Runner) only when VM crash which wasn't ideal. If you think this commit shouldn't be in this PR, I can create another PR. Would you prefer that?

Reply:
To me, this PR is fine, the rebase+merge result will be the same. Just wasn't sure how this commit is related.

A better commit description could have helped. ;-)